packages/components/src/components/modal/modal-styles.ts

```diff
@@ -145,7 +145,7 @@ export const getComponentCss = (
         boxSizing: 'border-box',
         transition: `transform .6s ${transitionTimingFunction}`,
         transform: open ? 'scale3d(1,1,1)' : 'scale3d(.9,.9,1)',
-        padding: !disableCloseButton ? `${pxToRemWithUnit(32)} 32px 32px 32px` : '32px', // rem value needed to prevent overlapping of close button and contents in scaling mode
+        padding: hasDismissButton ? `${pxToRemWithUnit(32)} 32px 32px 32px` : '32px', // rem value needed to prevent overlapping of close button and contents in scaling mode
```

Comment:
We could use here `spacingStaticLarge` variable

And also maybe `spacingFluidLarge` instead of `pxToRemWithUnit(32)`

Reply:
The use of these style variables should be improved across all components and we wouldn't start doing this as part of this PR. It's one of the cross cutting improvements like the focus and hover styles which will be applied in a clean up step once all components are v3 ready.